microproxy/cert.py

```diff
@@ -58,3 +58,13 @@ def get_cert_from_cache(self, common_name):
             return self.certs_cache[common_name]
         except KeyError:
             return None
+
+_cert_store = None
+
+
+def get_cert_store(config=None):
+    global _cert_store
+    if _cert_store:
+        return _cert_store
+    _cert_store = CertStore(config)
+    return _cert_store
```

Comment:
Is this function will make confused that when we pass different config, will get different cert store, but will get the same cert store.

I know it would not be a problem, but just provide my concern and have a discussion on it!